system/Session/Session.php

```diff
@@ -317,35 +379,35 @@ public function set($data, $value = null)
 	//--------------------------------------------------------------------
 
 	/**
-	 * Get any user data that has been set in the session.
+	 * Get user data that has been set in the session.
 	 *
+	 * If the property exists as "normal", returns it.
+	 * Otherwise, returns an array of any temp or flash data values with the
+	 * property key.
+	 * 
 	 * Replaces the legacy method $session->userdata();
 	 *
-	 * @param null $key
-	 *
-	 * @return array|null
+	 * @param  $key	Identifier of the session property to retrieve
+	 * @return array|null	The property value(s)
 	 */
 	public function get($key = null)
 	{
 		if (isset($key))
 		{
 			return isset($_SESSION[$key]) ? $_SESSION[$key] : null;
-		}
-		elseif (empty($_SESSION))
+		} elseif (empty($_SESSION))
```

Comment:
> The keyword `elseif` SHOULD be used instead of `else if` so that all control keywords look like single words.

http://www.php-fig.org/psr/psr-2/#5-1-if-elseif-else

It's a chicken/egg deal, but I think most of the people who have come to the PHP community since PSR-1/2 became well-known have adopted most of the ideas in both documents, even if almost every developer has their own pet exceptions. I also think most people attempting to write about best practices in PHP would be crazy to ignore PSR-1/2, even if they don't agree with them.